pkg/logutil/log.go

```diff
@@ -23,10 +23,10 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/coreos/etcd/raft"
 	"github.com/coreos/pkg/capnslog"
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
+	"go.etcd.io/etcd/raft"
```

Comment:
Why "github.com/coreos/etcd" is changed to "go.etcd.io/etcd" ? @nolouch

Reply:
`github.com/coreos/etcd` has renamed to use package name `go.etcd.io/etcd`